app/code/Magento/QuoteGraphQl/etc/schema.graphqls

```diff
@@ -102,6 +103,24 @@ input ShippingMethodForAddressInput {
     method_code: String!
 }
 
+input SetPaymentMethodOnCartInput {
+    cart_id: String!
+    payment_method: PaymentMethodInput!
+}
+
+input PaymentMethodInput {
+    code: String! @doc(description:"Payment method code")
+    po_number: String! @doc(description:"Purchase order number")
```

Comment:
It is better to use full field name "purchase_order_number". Is there a reason to shorten it here?